### ISSUE TITLE
Velocity startup error

### ISSUE THREAD

**Issue by x1p** [AlexProgrammerDE/PistonMOTD#211]
I'm tyring to make a setup for a Velocity proxy. Right from the initial startup, I get this:
```
[20:57:00 INFO] [pistonmotd]: _____ _ _ __ __ ____ _______ _____
[20:57:00 INFO] [pistonmotd]: | __ \(_) | | | \/ | / __ \|__ __|| __ \
[20:57:00 INFO] [pistonmotd]: | |__) |_ ___ | |_ ___ _ __ | \ / || | | | | | | | | |
[20:57:00 INFO] [pistonmotd]: | ___/| |/ __|| __|/ _ \ | '_ \ | |\/| || | | | | | | | | |
[20:57:00 INFO] [pistonmotd]: | | | |\__ \| |_| (_) || | | || | | || |__| | | | | |__| |
[20:57:00 INFO] [pistonmotd]: |_| |_||___/ \__|\___/ |_| |_||_| |_| \____/ |_| |_____/
[20:57:00 INFO] [pistonmotd]:
[20:57:00 INFO] [pistonmotd]: Loading config
[20:57:00 INFO] [pistonmotd]: Registering placeholders
[20:57:00 INFO] [pistonmotd]: Looking for hooks
[20:57:00 INFO] [pistonmotd]: Hooking into LuckPerms
[20:57:00 INFO] [pistonmotd]: Registering listeners
[20:57:00 INFO] [pistonmotd]: Registering command
[20:57:00 INFO] [pistonmotd]: Checking for a newer version
[20:57:00 ERROR]: Couldn't pass ProxyInitializeEvent to pistonmotd
java.lang.NumberFormatException: For input string: "{{ version }}"
at java.lang.NumberFormatException.forInputString(NumberFormatException.java:67) ~[?:?]
at java.lang.Integer.parseInt(Integer.java:668) ~[?:?]
at java.lang.Integer.parseInt(Integer.java:786) ~[?:?]
at net.pistonmaster.pistonmotd.shadow.pistonutils.update.UpdateParser$CurrentVersion.<init>(UpdateParser.java:67) ~[?:?]
at net.pistonmaster.pistonmotd.shadow.pistonutils.update.UpdateParser.parseUpdate(UpdateParser.java:19) ~[?:?]
at net.pistonmaster.pistonmotd.shared.PistonMOTDPlugin.lambda$checkUpdate$1(PistonMOTDPlugin.java:140) ~[?:?]
at net.pistonmaster.pistonmotd.shadow.pistonutils.update.UpdateChecker.getVersion(UpdateChecker.java:26) ~[?:?]
at net.pistonmaster.pistonmotd.shared.PistonMOTDPlugin.checkUpdate(PistonMOTDPlugin.java:140) ~[?:?]
at net.pistonmaster.pistonmotd.velocity.PistonMOTDVelocity.onProxyInitialization(PistonMOTDVelocity.java:65) ~[?:?]
at net.pistonmaster.pistonmotd.velocity.Lmbda$7.execute(Unknown Source) ~[?:?]
at com.velocitypowered.proxy.event.UntargetedEventHandler$VoidHandler.lambda$buildHandler$0(UntargetedEventHandler.java:56) ~[velocity.jar:3.2.0-SNAPSHOT (git-19abb909-b265)]
at com.velocitypowered.proxy.event.VelocityEventManager.fire(VelocityEventManager.java:597) ~[velocity.jar:3.2.0-SNAPSHOT (git-19abb909-b265)]
at com.velocitypowered.proxy.event.VelocityEventManager.lambda$fire$5(VelocityEventManager.java:478) ~[velocity.jar:3.2.0-SNAPSHOT (git-19abb909-b265)]
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
at java.lang.Thread.run(Thread.java:833) [?:?]
```
It seems to work, as far as I can see now

PistonMOTD 5.1.0
Velocity 3.2.0 b265

Version 5.0.1 works without errors

**Comment by AlexProgrammerDE** [comment]
Ah okay. I will make a fix tomorrow.

**Comment by AlexProgrammerDE** [comment]
Resolved by https://github.com/AlexProgrammerDE/PistonMOTD/commit/c1abdb6ab3253cbec499224eea08eacd23971409
The fix has been released in `5.1.1`
